specs-go/v1/mediatype.go

```diff
@@ -70,6 +70,6 @@ const (
 	// MediaTypeImageConfig specifies the media type for the image configuration.
 	MediaTypeImageConfig = "application/vnd.oci.image.config.v1+json"
 
-	// MediaTypeScratch specifies the media type for an unused blob containing the value `{}`
-	MediaTypeScratch = "application/vnd.oci.scratch.v1+json"
+	// MediaTypeFiller specifies the media type for an unused blob containing the value `{}`
+	MediaTypeFiller = "application/vnd.oci.filler.v1+json"
```

Comment:
Maybe:
```suggestion
	MediaTypeFiller = "application/vnd.oci.image.filler.v1+json"
```

Reply:
Looking over the full list again, this does feel like a more consistent choice with our existing media types, but I don't feel really strongly either way. :+1:

Reply:
I don't feel strongly on this either. It's more consistent to call it "image" since it's defined in the image spec and I don't want to step across to other specs. But if we start to define media types in the image-spec that are not an "image", then this new media type feels like it spans use cases.